### DISCUSSION TITLE
Need Assistance Deploy AAD Pod Identity

### DISCUSSION THREAD

**Discussion by cramsey86** [Azure/aad-pod-identity#1031] [Q&A] [answered]
Hello,

We have aad-pod-identity deployed to all of our AKS clusters. 

Today we had to rebuild an AKS cluster, however, when installing aad-pod-identity we are getting the following error on the pods starting up:

``
53m         Warning   Failed              pod/aad-pod-identity-mic-7679b647c8-24fmx    Failed to pull image "mcr.microsoft.com/oss/azure/aad-pod-identity/mic:v1.7.5": rpc error: code = NotFound desc = failed to pull and unpack image "mcr.microsoft.com/oss/azure/aad-pod-identity/mic:v1.7.5": failed to resolve reference "mcr.microsoft.com/oss/azure/aad-pod-identity/mic:v1.7.5": mcr.microsoft.com/oss/azure/aad-pod-identity/mic:v1.7.5: not found
``

For both the nmi and the mic. 

The AKS Cluster is v1.19.7.

We are able to resolve mcr.microsoft.com from the cluster.

Commands run to install were:

``
sudo helm repo add aad-pod-identity https://raw.githubusercontent.com/Azure/aad-pod-identity/master/ch
arts
``
``
sudo helm install --namespace aad-pod-identity rsm-aad-pod-identity --wait aad-pod-identity/aad-pod-id
entity
``

Any suggestions would be greatly appreciated.

Thanks

**Comment by aramase** [comment, marked as answer]
@cramsey86 The new release was being published. It's now available.
```bash
➜ docker manifest inspect upstream.azurecr.io/oss/azure/aad-pod-identity/mic:v1.7.5
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:6652c3035e2a4664af66c0de812de0ac6458bcf0726cd3ba716d0120c7376c99",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:a6a7e2163ba0aaf9a0fb523a9f19126289ace12e5c70f593e568c098f2426c90",
         "platform": {
            "architecture": "arm",
            "os": "linux",
            "variant": "v7"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:de078a7225b43f4f7870ae7bdac3d65a89a17455e80bf35d0bf66f3dfe77b63e",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```
Please retry the install command.

**Reply by cramsey86** [reply]
HA! Talk about timing! They are functional! Thank YOU!!!
